client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -169,19 +169,23 @@ export const getTask = (
 				timing: 10,
 				title: translate( 'Create a site menu' ),
 				description: translate(
-					"Building an effective navigation menu makes it easier for someone to find what they're looking for and improve search engine rankings."
+					"Building an effective navigation menu makes it easier for someone to find what they're looking for and improve search engine rankings. {{supportLink/}}.",
```

Comment:
Could we concat the link and the rest of text, so we don't change the existing translatable string?

```
description: (
	<>
		{ translate(
			"Building an effective navigation menu makes it easier for someone to find what they're looking for and improve search engine rankings."
		) }
		<InlineSupportLink
			supportPostId={ 59580 }
			supportLink={ localizeUrl( 'https://wordpress.com/support/menus/' ) }
			showIcon={ false }
			text={ translate( 'View tutorial.' ) }
		/>
	</>
),
```

That way translators won't need to re-translate the task description and we'll continue re-using the existing translation. 

Note that with this approach the dot needs to be included as part of the translation (`translate( 'View tutorial.' )`), but that's ok, since our i18n tooling is smart enough with these one single characters changes and will continue using the existing translation while the new ones are translated.

It will also need to separate the link with some CSS, but we already have some prior art for that:

https://github.com/Automattic/wp-calypso/blob/7e5c83c7eb3c9da2a76f23e5745231e8fbb43ef7/client/my-sites/customer-home/cards/features/stats-v2/style.scss#L39-L41

(It could be also done with a `nbsp`, but @gwwar is [not a big fan of them](https://github.com/Automattic/wp-calypso/pull/41441#discussion_r416155575) 😅).

Reply:
Yeah, that makes sense, fixed! I used a whitespace because it seemed simpler than adding a class name to the tasks (though I think it can be done with `currentTask.id`), but let me know if this isn't ideal!

Reply:
> I used a whitespace

That works too 👍